kani-driver/src/main.rs

```diff
@@ -66,6 +66,11 @@ fn cargokani_main(input_args: Vec<OsString>) -> Result<()> {
 
     let mut failed_harnesses: Vec<&HarnessMetadata> = Vec::new();
 
+    if ctx.args.print_harness_metadata {
+        ctx.return_list_harnesses(&sorted_harnesses).unwrap();
```

Comment:
BTW, can you print the entire metadata here?

Reply:
I wouldn't print the whole metadata yet.

What we currently do to "merge" all metadata files into one if not a good idea, it just happens to work right now. (For instance, "assess" will add metadata that, when merged, should not turn into a flat vector, but a hash map of "crate -> unsupported features".)

I recommend starting with just the actually needed information here, because otherwise it'll create complications as we evolve the metadata's structure.